WORKSPACE

```diff
@@ -230,6 +230,13 @@ go_repository(
     version = "v2.2.6",
 )
 
+go_repository(
+    name = "com_github_osrg_gobgp_v3",
+    importpath = "github.com/osrg/gobgp/v3",
+    sum = "h1:zfXhM8+QSfAPUkzWYTbH8pMkvxkcUfvQEKnBU7ikk4g=",
```

Comment:
don't add go_repository to this file, add them to reposirories.bzl

Reply:
Updated Makefile to write to this file.